howso/engine/trainee.py

```diff
@@ -148,6 +149,7 @@ def __init__(
             library_type=library_type,
             max_wait_time=max_wait_time,
             overwrite=overwrite_existing,
+            replicas=replicas,
```

Comment:
Need to use runtime in this file as well.

Reply:
Fixed.